### DISCUSSION TITLE
Zero (0) speed not supported any more?

### DISCUSSION THREAD

**Discussion by TestDmitriy** [Rem0o/FanControl.Releases#2406] [Q&A]
Hi Rem0o,
For some reasons after one of the last updates program doesn't support sero(0%) speed of my 2 SYS fans ( 2 fans Corsair 140mm).
Before my both fans stopped at low temp. In BIOS both sys fans sat at MIN speed settings. 
So, FanControl controls only speed higher than BIOS settings and doesn't override BIOS min speed settings like used to before. 
*I have MSI MAG z690m mobo with latest bios update.
When i tried to pair fans after last update to v. 185, program can't recognize my 2 sys fans at all :( 
I returned to 181 version of program, version w/t new calibration rpm.

Thank you.

**Comment by TestDmitriy** [comment]
Just figured it out, 
Solution for - Ho to make fans run at 0% rpm
After "playing" with fan settings in BIOS - i switched both my Sys fans to DC mode and it works now.
PWM mode doesn't let FanControl "stop your fans at 0 rpm" it will always run at min speed only.

*Would be great to add that info to any kind of help or general info about  program settings.

Thank you,

**Reply by QGtKMlLz** [reply]
Darn, I was really hoping there was a PWM mode way.
